src/main/java/com/yugabyte/sample/common/CmdLineOpts.java

```diff
@@ -146,7 +149,7 @@ public void initialize(CommandLine commandLine) throws ClassNotFoundException {
     }
     LOG.info("Local reads: " + localReads);
     LOG.info("Read only load: " + readOnly);
-    if (appName == AppName.CassandraBatchTimeseries) {
+    if (appName == CassandraBatchTimeseries.class.getSimpleName()) {
```

Comment:
Do we need `appName.equals(CassandraBatchTimeseries.class.getSimpleName())` for this and other cases below?